### PR TITLE
Implemented Update and Delete Posts

### PR DESCRIPTION
**Pull request by wDushanin**
Implemented Update and Delete Posts

### CODE REVIEW

File: petpals/blueprints/post_blueprint.py [hunk 1]
```diff
@@ -32,3 +33,64 @@ def post_form():
...
+        return redirect(url_for('forum', post_id=post.post_id))
+
+    if current_user.id == post.user_id or current_user.id:
+        form.title.data = post.title
+        form.content.data = post.content
+        return render_template('edit_post.html', form=form)
+    else:
+        flash("You Aren't Authorized To Edit This Post...")
+        posts = Post.query.order_by(Post.timestamp)
+        return render_template("forum.html", posts=posts)
```
Comment (EthanPintoA) [inline petpals/blueprints/post_blueprint.py:63]: Should be a redirect

File: petpals/blueprints/post_blueprint.py [hunk 2]
```diff
@@ -32,3 +33,64 @@ def post_form():
...
+
+@router.route('/posts/delete/<int:post_id>')
```
Comment (EthanPintoA) [inline petpals/blueprints/post_blueprint.py:65]: Shouldn't have `/posts`

File: petpals/blueprints/post_blueprint.py [hunk 3]
```diff
@@ -32,3 +33,64 @@ def post_form():
...
+		try:
+			db.session.delete(post_to_delete)
+			db.session.commit()
+
+			# Return a message
+			flash("Blog Post Was Deleted!", 'success')
+
+			# Grab all the posts from the database
+			posts = Post.query.order_by(Post.timestamp)
+			return render_template("forum.html", posts=posts)
```
Comment (EthanPintoA) [inline petpals/blueprints/post_blueprint.py:80]: Should be a redirect

File: petpals/blueprints/post_blueprint.py [hunk 4]
```diff
@@ -32,3 +33,64 @@ def post_form():
...
+
+
+		except:
+			# Return an error message
+			flash("Whoops! There was a problem deleting post, try again...")
+
+			# Grab all the posts from the database
+			posts = Post.query.order_by(Post.timestamp)
+			return render_template("posts.html", posts=posts)
```
Comment (EthanPintoA) [inline petpals/blueprints/post_blueprint.py:89]: `posts.html` does not exit. Should be a redirect to post.html with a post_id of `post_id`

File: petpals/blueprints/post_blueprint.py [hunk 5]
```diff
@@ -32,3 +33,64 @@ def post_form():
...
+	else:
+		# Return a message
+		flash("You Aren't Authorized To Delete That Post!", 'danger')
+
+		# Grab all the posts from the database
+		posts = Post.query.order_by(Post.timestamp)
+		return render_template("forum.html", posts=posts)
```
Comment (EthanPintoA) [inline petpals/blueprints/post_blueprint.py:96]: Should be a redirect

**Review by EthanPintoA — CHANGES_REQUESTED** [review]

**Comment by wDushanin** [comment]
@EthanPintoA All requested changes have been committed.

**Comment by EthanPintoA** [comment]
Moved to #74

**Pull request closed without merging**